docs/how_to/create_custom_map.rst

```diff
@@ -0,0 +1,267 @@
+.. _how_to_custom_maps:
+
+Create Custom Maps by Hand
```

Comment:
```suggestion
Create custom Maps by hand
```
I do not like capitalizing words but maybe this is worth worrying about if we ever form a style guide?

Reply:
I've followed this same capitalization for all other sections (e.g. in the tutorial) so I'd rather have this debate later on when we either do a largescale style refactor and/or write a style guide.